pkg/types/block_state.go

```diff
@@ -209,6 +262,13 @@ func (b *StateBlock) Clone() *StateBlock {
 	return &clone
 }
 
+func (b *StateBlock) IsPrivate() bool {
+	if len(b.PrivateFrom) > 0 {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
S1008: should use 'return <expr>' instead of 'if <expr> { return <bool> }; return <bool>' (gosimple)